NEWS

```diff
@@ -1,3 +1,26 @@
+Airlift 0.109
+
+* Default Config
+
+An application typically need to override the default configuration of some components.
+For example, the default timeouts in the HTTP client are too large for communication
+between workers in a data center.  These applications need to ship with a default
```

Comment:
"Rather than requiring these applications to ship with a default configuration file, we added ..."